src/platform/forms-system/src/js/components/ContactInfo.jsx

```diff
@@ -0,0 +1,397 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+import { Element } from 'react-scroll';
+
+import {
+  focusElement,
+  scrollTo,
+  scrollAndFocus,
+} from '@department-of-veterans-affairs/platform-utilities/ui';
+import environment from '@department-of-veterans-affairs/platform-utilities/environment';
+
+import {
+  selectProfile,
+  isLoggedIn,
+} from '@department-of-veterans-affairs/platform-user/selectors';
+
+// import { generateMockUser } from 'platform/site-wide/user-nav/tests/mocks/user';
+import { generateMockUser } from '../../../../site-wide/user-nav/tests/mocks/user';
+
+// import { AddressView } from '@department-of-veterans-affairs/platform-user/exports';
+// import AddressView from '@@vap-svc/components/AddressField/AddressView';
+import AddressView from '../../../../user/profile/vap-svc/components/AddressField/AddressView';
+
+// import FormNavButtons from '@department-of-veterans-affairs/platform-forms-system/FormNavButtons';
+// import FormNavButtons from 'platform/forms-system/src/js/components/FormNavButtons';
+import FormNavButtons from './FormNavButtons';
+
+import readableList from '../utilities/data/readableList';
+import {
+  getReturnState,
+  clearReturnState,
+  getPhoneString,
+  getMissingInfo,
+  REVIEW_CONTACT,
+  contactInfoPropTypes,
+} from '../utilities/data/profile';
+
+/**
+ * Render contact info page
+ * @param {Object} data - full form data
+ * @param {Function} goBack - CustomPage param
+ * @param {Function} goForward - CustomPage param
+ * @param {Boolean} onReviewPage - CustomPage param
+ * @param {Function} updatePage - CustomPage param
+ * @param {Element} contentBeforeButtons - CustomPage param
+ * @param {Element} contentAfterButtons - CustomPage param
+ * @param {Function} setFormData - CustomPage param
+ * @param {Object} content - Contact info page content
+ * @param {String} contactPath - Contact info path; used in edit page path
+ * @parma {import('../utilities/data/profile').ContactInfoKeys} keys - contact info data key
+ * @param {String[]} requiredKeys - list of keys of required fields
+ * @returns
+ */
+const ContactInfo = ({
+  data,
+  goBack,
+  goForward,
+  onReviewPage,
+  updatePage,
+  contentBeforeButtons,
+  contentAfterButtons,
+  setFormData,
+  content,
+  contactPath,
+  keys,
+  requiredKeys,
+}) => {
+  const wrapRef = useRef(null);
+  window.sessionStorage.setItem(REVIEW_CONTACT, onReviewPage || false);
+  const [hadError, setHadError] = useState(false);
+  const [submitted, setSubmitted] = useState(false);
+  const [editState] = useState(getReturnState());
+
+  // vapContactInfo is an empty object locally, so mock it
+  const profile = useSelector(selectProfile) || {};
+  const loggedIn = useSelector(isLoggedIn) || false;
+  const contactInfo =
+    loggedIn && environment.isLocalhost()
+      ? generateMockUser({ authBroker: 'iam' }).data.attributes
+          .vet360ContactInformation
+      : profile.vapContactInfo || {};
```

Comment:
Yeah, locally the `profile.vapContactInfo` is an empty object so to get past the page, when there are required contact info fields, I need to populate the form data with something... I know it's not ideal, but it only does this locally, so that's a plus, right? LOL